packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -640,7 +698,7 @@ export class BiconomySmartAccountV2 extends BaseSmartContractAccount {
    * @param buildUseropDto options for building the userOp
    * @returns Promise<UserOpResponse>
    */
-  async sendTransaction(manyOrOneTransactions: Transaction | Transaction[], buildUseropDto?: BuildUserOpOptions) {
+  async sendTransaction(manyOrOneTransactions: Transaction | Transaction[], buildUseropDto?: BuildUserOpOptions): Promise<UserOpResponse> {
```

Comment:
just adding a note here,

1. options to do either from below,
dummy pnd this.getDummyPnd(). paymaster.getdummypnD(). override paymasterAndData

2. making use of supplied overrides
only for bundler estimations (can be additional param in bundler.estimateUserOpGas without overriding userOp)

potentially adding a new method buildSponsoredUserOpOptimised() which will rely on calling paymaster only for gas estimations and expects paymasterAndData return.

Reply:
This is for next iteration yes ?

Reply:
yes can be separate PR